docs/admin/install.rst

```diff
@@ -63,11 +63,11 @@ install them you can use apt-get:
         python-whoosh python-pil python-libravatar \
         python-babel Git mercurial python-social-auth \
         python-django-compressor python-django-crispy-forms \
-        python-dateutil
+        python-djangorestframework python-dateutil
 
     # Optional for database backend
 
-    apt-get install python-mysqldb   # For MySQL
+    apt-get install python-pymysql   # For MySQL
```

Comment:
I got this on Ubuntu 16.04 with python-mysqldb installed:

`Traceback (most recent call last):
  File "/var/www/weblate/weblate/wsgi.py", line 44, in <module>
    application = get_wsgi_application()
  File "/usr/lib/python2.7/dist-packages/django/core/wsgi.py", line 14, in get_wsgi_application
    django.setup()
  File "/usr/lib/python2.7/dist-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/lib/python2.7/dist-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/usr/lib/python2.7/dist-packages/django/apps/config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/django/contrib/auth/models.py", line 41, in <module>
    class Permission(models.Model):
  File "/usr/lib/python2.7/dist-packages/django/db/models/base.py", line 139, in __new__
    new_class.add_to_class('_meta', Options(meta, **kwargs))
  File "/usr/lib/python2.7/dist-packages/django/db/models/base.py", line 324, in add_to_class
    value.contribute_to_class(cls, name)
  File "/usr/lib/python2.7/dist-packages/django/db/models/options.py", line 250, in contribute_to_class
    self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
  File "/usr/lib/python2.7/dist-packages/django/db/__init__.py", line 36, in __getattr__
    return getattr(connections[DEFAULT_DB_ALIAS], item)
  File "/usr/lib/python2.7/dist-packages/django/db/utils.py", line 241, in __getitem__
    backend = load_backend(db['ENGINE'])
  File "/usr/lib/python2.7/dist-packages/django/db/utils.py", line 112, in load_backend
    return import_module('%s.base' % backend_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/dist-packages/django/db/backends/mysql/base.py", line 23, in <module>
    import pymysql
`

After installing python-pymysql it worked